### PR TITLE
[GITHUB-1] Add initial role

### PR DESCRIPTION
**Pull request by lobsterdore**

### CODE REVIEW

**Review by dennisconrad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,236 +1,74 @@
...
-ANSIBLE_PIP_VERSIONS=`curl -s https://pypi.org/pypi/ansible/json`; for I in 2.{3..5}; do echo ${ANSIBLE_PIP_VERSIONS} | jq --arg ver ${I} -r '[.releases | keys | sort[] | select(startswith($ver))][-1]'; done
+and run `ansible-galaxy install -p ./roles -r roles.yml`
+
+## Role Variables
+
+|Variable|Default|Description|
+|---|---|---|
+|sansible_aws_vpc_cidr|~|CIDR for VPC|
+|sansible_aws_vpc_cidr_subnet_private_a|~|CIDR for private subnet a|
+|sansible_aws_vpc_cidr_subnet_private_b|~|CIDR for private subnet a|
```
Comment (nathanwebsterdotme) [inline README.md]: "CIDR for private subnet a" should be "CIDR for private subnet b"

File: README.md [hunk 2]
```diff
@@ -1,236 +1,74 @@
...
+|sansible_aws_vpc_cidr_subnet_private_c|~|CIDR for private subnet a|
```
Comment (nathanwebsterdotme) [inline README.md]: "CIDR for private subnet a" should be "CIDR for private subnet c"

File: README.md [hunk 3]
```diff
@@ -1,236 +1,74 @@
...
+|sansible_aws_vpc_cidr_subnet_public_a|~|CIDR for public subnet a|
+|sansible_aws_vpc_cidr_subnet_public_b|~|CIDR for public subnet a|
```
Comment (nathanwebsterdotme) [inline README.md]: "CIDR for public subnet a" should be "CIDR for public subnet b"

File: README.md [hunk 4]
```diff
@@ -1,236 +1,74 @@
...
+|sansible_aws_vpc_cidr_subnet_public_c|~|CIDR for public subnet a|
```
Comment (nathanwebsterdotme) [inline README.md]: "CIDR for public subnet a" should be "CIDR for public subnet c"

File: tasks/main.yml [hunk 5]
```diff
@@ -1 +1,60 @@
 ---
+
+- name: Assert configuration
+  assert:
+    that:
+      - sansible_aws_vpc_cidr is not none and sansible_aws_vpc_cidr | length > 0
+      - sansible_aws_vpc_cidr_subnet_private_a and sansible_aws_vpc_cidr_subnet_private_a | length > 0
```
Comment (nathanwebsterdotme) [inline tasks/main.yml]: Do we want to force always having to use AZ a?  What if I only want a single subnet it AZ c?

Comment (lobsterdore) [reply tasks/main.yml]: Yeah this is intentional to make all the conditionals work in the template, if someone needs such a setup then we can revisit this.

**Review by nathanwebsterdotme — APPROVED** [review]

**Pull request merged**